CMakeLists.txt

```diff
@@ -127,7 +127,9 @@ if(APPLE)
 #  set(DYLIB_COMPAT_VERSION 12.0.0 CACHE STRING "library compatibility version")
 #  set(DYLIB_CURRENT_VERSION 12.61.0 CACHE STRING "library current version")
   find_package(X11)
-  include_directories("${X11_INCLUDE_DIR}")
+  if(X11_FOUND)
+    include_directories("${X11_INCLUDE_DIR}")
```

Comment:
Do you hit `X11_FOUND` with `X11_INCLUDE_DIR` set to something non-empty?

Reply:
You should not: https://github.com/Kitware/CMake/blob/f72c405d4ee4ad4775a10a944774b91f49726dad/Modules/FindX11.cmake#L469-L471

Reply:
Yes, and that's what I'm saying: `X11_INCLUDE_DIR` is wrong here. It should be `X11_X11_INCLUDE_PATH`, hoping that name it won't change across cmake versions.

Reply:
To be more clear: You are _not_ using `FindX11` module which should set `X11_INCLUDE_DIR`. Instead, you are using `find_package()`.

Reply:
But `X11_INCLUDE_DIR` is documented at [`FindX11`](https://cmake.org/cmake/help/v3.0/module/FindX11.html)?

`find_package(X11)` looks for `FindX11.cmake` or `X11Config.cmake` files and includes these.

Reply:
[The search procedure is documented here](https://cmake.org/cmake/help/latest/command/find_package.html#search-modes)

Reply:
OK then.